tensilelite/Tensile/SolutionStructs.py

```diff
@@ -1917,9 +1918,9 @@ def isDirectToVgprDoable(state, tc):
       reject(state, "DirectToVgpr%c does not supports InnerUnroll>1"%(tc))
       return False
 
-    # Reject TLU = UnrollMajorLDS (B only)
-    if tc == 'B' and (state["ProblemType"]["TLUA"] == state["UnrollMajorLDSA"] or state["ProblemType"]["TLUB"] == state["UnrollMajorLDSB"]):
-      reject(state, "DirectToVgpr%c does not supports TLU = UnrollMajorLDS"%(tc))
+    # Reject TLU = UnrollMajorLDS (Either for DTVA/B)
+    if (state["ProblemType"]["TLUA"] == state["UnrollMajorLDSA"]) or (state["ProblemType"]["TLUB"] == state["UnrollMajorLDSB"]):
```

Comment:
should check the condition by referring to tc

Reply:
Done, tests passed